sonar-plugin-api/src/main/java/org/sonar/api/batch/bootstrap/ProjectDefinition.java

```diff
@@ -442,40 +284,11 @@ public ProjectDefinition setTests(File... fileOrDirs) {
     return this;
   }
 
-  /**
-   * @deprecated since 4.5 use {@link #setTests(File...)}
-   */
-  @Deprecated
-  public ProjectDefinition setTestDirs(File... dirs) {
-    return setTests(dirs);
-  }
-
   public ProjectDefinition resetTests() {
     properties.remove(TESTS_PROPERTY);
     return this;
   }
 
-  /**
-   * @deprecated since 4.5 use {@link #resetTests()}
-   */
-  @Deprecated
-  public ProjectDefinition resetTestDirs() {
-    return resetTests();
-  }
-
-  /**
-   * @deprecated since 4.5 use {@link #addTests(String...)}
-   */
-  @Deprecated
-  public ProjectDefinition addTestFiles(String... paths) {
-    // Hack for visual studio project builder that used to add baseDir first as test dir
-    List<String> testDirs = getTestDirs();
-    if (testDirs.size() == 1 && isDirectory(testDirs.get(0))) {
-      resetTests();
-    }
-    return addTests(paths);
-  }
-
   private boolean isDirectory(String relativeOrAbsoluteDir) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this unused private "isDirectory" method. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/sonarqube/coding_rules#rule_key=squid%3AUnusedPrivateMethod)